### DISCUSSION TITLE
Feature request: mod_articles_category filter featured articles improvement

### DISCUSSION THREAD

**Discussion by Notebit** [joomla/joomla-cms#29278] [2. Cold Features]
### Steps to reproduce the issue
mod_articles_category is often used for displaying articles on standard joomla homepages created with featured articles blog list.
For that reason is very useful that mod_articles_category allows to filter featured articles and eventually hide featured articles.

It would be even better if we can set a number of featured articles to hide, cause most of the time hiding all of them is not what we want, we just want to hide an amount of featured articles which is already displayed on the homepage. So if we show 20 featured articles in homepage, we can still use the mod_articles_category in homepage, hiding 20 featured articles and avoiding duplicated content in homepage.

**Comment by brianteeman** [comment]
Why would you want to "hide a featured article" instead of removing its featured status

**Comment by Notebit** [comment]
cause we use featured articles for the homepage for building the homepage, an homepage on more pages, at the same time we use mod_articles_category and other com_content modules for making the homepage a bit richer, but we display them just on the homepage itself not on the secondary pages.
So we need to keep featured articles tag, but we need to avoid mod_articles_category from showing duplicated content in homepage. It is a very common issue with joomla, how to avoid duplicated content in homepage when you display several blocks of contents, and filtering mod_articles_category and other com_content articles for hiding a certain amount of featured articles would solve the problem.
Let's say I show 15 featured articles by page in featured articles homepage, I just need to hide 15 featured articles to mod_articles_category and I don't have the issue anymore. Of course there are more complex and refined ways to solve this issue, but I think this one can be quite effective and simple. But if there are better ways to solve the homepage duplicated issues just using standard joomla features, please let me know.

For clarifying: I want mod_articles_category to hide just the first N featured articles, so the idea is to add the possibility to limit to N the hide featured articles feature already supported by the module<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29278">issues.joomla.org/tracker/joomla-cms/29278</a>.</sub>
